common/persistence/nosql/nosqlplugin/cassandra/workflow.go

```diff
@@ -398,14 +398,23 @@ func (db *cdb) SelectTransferTasksOrderByTaskID(ctx context.Context, shardID, pa
 		}
 	}
 
-	var tasks []*nosqlplugin.TransferTask
+	var tasks []*nosqlplugin.HistoryMigrationTask
 	task := make(map[string]interface{})
 	for iter.MapScan(task) {
 		t := parseTransferTaskInfo(task["transfer"].(map[string]interface{}))
+		taskID := task["task_id"].(int64)
+		data := task["data"].([]byte)
+		encoding := task["data_encoding"].(string)
```

Comment:
Doesn't this cause panic when data_encoding is not present? I guess populates the map for columns specified in select clause even if they are empty/nil

Reply:
I tested it in dev2 and I didn't see panics.